streambed-logged/src/lib.rs

```diff
@@ -491,6 +492,17 @@ impl CommitLog for FileLog {
                         topic_file_op
                     };
 
+                    let modification_time = topic_file_op.modification_time();
+                    if modification_time <= last_modification_time {
```

Comment:
Is this check correct when either time value is None?

Reply:
Yes. :-)

mod time is none, last mod time is none, there are no files so we sleep.
mod time is none, last mod time is some, there are no files so we sleep (someone removed the files!)
mod time is some, last mod time is none, there are files so need to process.
mod time is some, last mod time is some, there are files so we compare times to see if we need to process.